### PR TITLE
Fixing error translator.

### PR DESCRIPTION
**Pull request by elbing**
Change-Id: Ie33fa22ecbb25da8c0a0d25f3f5bbf12c7eb4c2a

### CODE REVIEW

**Comment by elbing** [comment]
I'm sorry for title, some character broke hub tool for pull-requests :-(

**Comment by rminnich** [comment]
The errno list should be in an architecture independent directory, but we can fix this later if you want. Nice job.

File: amd64/include/bits/errno.h [hunk 1]
```diff
@@ -0,0 +1,134 @@
+#define EPERM            1
```
Comment (rminnich) [inline amd64/include/bits/errno.h]: This file can be in include/ if you want. It's architecture independent.

**Comment by rminnich** [comment]
LGTM

but you can still move that file :-)

**Comment by elbing** [comment]
Rarely, it was under arch dependent dir in musl. I think it still can be an independent file in sys/errno.h BSD style. I'll move it before merging this.

Seeing /usr/include/bits/errno.h in linux, it seems to be a divergency with kernel headers about some (missing) defined errors. In fact, originally, EDOM/ERANGE/ were different in old APE that in POSIX implementation. But differences seems to be between OS'es, and not archs. That's why I'll move the file.

Thanks guys!

**Comment by elbing** [comment]
Moved to sys/errno.h.

**Pull request merged**